docs/BuildFromSource.md

```diff
@@ -58,22 +53,66 @@ git submodule update --init --recursive
 
 ## Building in Visual Studio / Code
 
-Before opening our .sln files in Visual Studio or VS Code, executing the following on command-line:
-```
-.\build.cmd /t:Restore
-```
-This will download required tools.
+Before opening our .sln files in Visual Studio or VS Code, you need to perform the following actions.
+
+1. Executing the following on command-line:
+   ```
+   .\build.cmd /p:SkipTests=true /p:_ProjectsOnly=true
```

Comment:
`/p:BuildNumberSuffix=t9999`, or have you addressed that another way?

Reply:
`/p:BuildNumberSuffix=t9999` is only necessary in the master branch, but should be resolved by #4311. I'll add that to docs when I merge this up thru 2.2 and master.